crosswalks/usgs-modelcatalog.csv

```diff
@@ -0,0 +1,74 @@
+Property,usgs-modelcatalog
+codeRepository,software.usgs_software_gitlab_release_urls / software.external_software_repository_urls
+programmingLanguage,programming_language
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,item_type
+applicationSubCategory,type_keywords
+downloadUrl,software.download_url
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,version
+storageRequirements,
+supportingData,
+author,person.name when person_type=Author
+citation,publications.model_citations
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,
+dateModified,last_update
+datePublished,release_date
+editor,
+encoding,
+fileFormat,
+funder,
+keywords,science_keywords / other_keywords
+license,license
+producer,organization / external_organization_name
+provider,
+publisher,
+sponsor,
+version,version
+isAccessibleForFree,
+isPartOf,related_modelcatalog_assets
+hasPart,related_modelcatalog_assets
```

Comment:
Hmm... since `related_modelcatalog_assets` doesn't map to either, it might be best to omit it entirely. (And there is also the possibility that a "related asset" is neither a superset not a part of the model being described, right?

Reply:
OK, I wasn't sure how to handle the case that the the thing in the field _might_ be a superset or part of the model, but might not be.  I can omit.